app/services/passporting/means_passporter.rb

```diff
@@ -5,7 +5,11 @@ def call
       return true if appeal_no_changes?
 
       means_passport = []
-      means_passport << MeansPassportType::ON_NOT_MEANS_TESTED if app_not_means_tested?
+
+      if app_not_means_tested? && FeatureFlags.non_means_tested.enabled?
+        means_passport << MeansPassportType::ON_NOT_MEANS_TESTED
+      end
```

Comment:
I think the feature flag could be a guard clause in the `app_not_means_tested` method, would be easier to follow